lib/carbon/client.py

```diff
@@ -2,8 +2,9 @@
 from time import time
 from six import with_metaclass
 
+from OpenSSL import SSL
 from twisted.application.service import Service
-from twisted.internet import reactor
+from twisted.internet import ssl, reactor
```

Comment:
Will it work on Twisted version < 14 ? 
If not - then we should raise min Twisted version in requirements.txt

Reply:
It does, however it does not include the tlsext hostname prior to 14. So, you still get a secure connection, but that TLS extension is not used..